dlpy/tests/test_network.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env python
+# encoding: utf-8
+#
+# Copyright SAS Institute
+#
+#  Licensed under the Apache License, Version 2.0 (the License);
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+# NOTE: This test requires a running CAS server.  You must use an ~/.authinfo
+#       file to specify your username and password.  The CAS host and port must
+#       be specified using the CASHOST and CASPORT environment variables.
+#       A specific protocol ('cas', 'http', 'https', or 'auto') can be set using
+#       the CASPROTOCOL environment variable.
+#
+
+import swat
+import swat.utils.testing as tm
+from dlpy.network import Network
+from dlpy.model import Model
+from dlpy.layers import *
+from dlpy.utils import DLPyError
+
+
+class TestNetwork(tm.TestCase):
+    # Create a class attribute to hold the cas host type
+    server_type = None
+    s = None
+
+    @classmethod
+    def setUpClass(cls):
+        swat.reset_option()
+        swat.options.cas.print_messages = False
+        swat.options.interactive_mode = False
+
+        cls.s = swat.CAS()
+        cls.server_type = tm.get_cas_host_type(cls.s)
+
```

Comment:
In a lot of the tests, I see things being run, but I don't see that any results are being checked.